packages/app-project/components/ClassifyPage/components/FinishedForTheDay/FinishedForTheDayContainer.js

```diff
@@ -0,0 +1,40 @@
+import { get } from 'lodash'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { inject, observer } from 'mobx-react'
+
+import FinishedForTheDay from './FinishedForTheDay'
+
+@inject('store')
+@observer
+class FinishedForTheDayContainer extends Component {
+  getComponentProps () {
+    const { props } = this
+    const projectName = get(props, 'store.project.displayName', '')
+    const imageSrc = get(props, 'store.project.backgrounds[0].src', '')
```

Comment:
Existential operator: I think lodash's `get` is easier to grok, since this request would require multiple `?`s scattered through the chain, but I've made a commit - see what you think. Fair enough on setting a default here.

Multiple backgrounds: They _could._ The `backgrounds` resource returned from the API is an array, so while the project builder only allows for one background to be uploaded at a time, it doesn't seem like a big deal to futureproof this.

Reply:
TBH, I'm not super familiar with lodash's API, so I tend to have to look it up what exactly it does when I come across it in the code. For that reason, I'd prefer to use what's spec or upcoming spec rather than rely on a library, though, we're already using lodash in other places and it's only stage one and not a critical feature like the stage one observable, so I don't have a strong opinion about this _yet_ 😉.  

I think all panoptes resource responses return in an array regardless of the modal association. There appears to be a model validation for only one background, but that is technically just stored in the media model as all media is, so, it seems maybe a matter of serialization and model validation whether or not there's one background? Projects have many media, but only one medium serialized as project background? I find this a bit confusing about the actual Panoptes database. https://github.com/zooniverse/Panoptes/blob/master/app/models/project.rb#L28

Reply:
In that case, I'll move this to the Project model, which is probably the correct place to do that anyway.